1-Initial-Setup/1.5-Additional-Process-Hardening.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+@test "1.5.1 Ensure XD/NX support is enabled (Manual)" {
+    run bash -c "journalctl | grep 'protection: active'"
+    [ "$status" -eq 0 ]
```

Comment:
We could test if the "protection: active"-string is included in the $output.
```suggestion
    [ "$status" -eq 0 ]
    [[ "$output" == *"NX (Execute Disable) protection: active"* ]]
```

Reply:
This is already done by the `grep` command. If the string is not included, the grep command will not exit with code 0.